common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-avoid-additional-properties-fps_2023-08-23-23-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Fix an issue where AvoidAdditionalProperties rule reported as FPS and this rule from now on will not check the additionalProperties defined from common-types or other predefined references.",
```

Comment:
>and [](http://example.com/codeflow?start=84&length=3)

Small sentences are more readable. So you can split this into 2 sentences.

Reply:
Modified and split the line into 2.